### ISSUE TITLE
[FEATURE REQUEST] 

### ISSUE THREAD

**Issue by iWisp360** [namidaco/namida#222]
**Is your feature request related to a problem? Please describe.**
The player only allows editing metadata lyrics by pasting the lyrics. I'd also want a synced lyrics editor.

**Additional context**
![Screenshot_2024-06-13-09-45-47-330_in krosbits musicolet](https://github.com/namidaco/namida/assets/114373500/eba485b1-0199-4610-8a1a-a14b59ceda34)

**Apps**
Musicolet

**Comment by MSOB7YY** [comment]
yeah uh maybe later, i considered it while implementing lrc view but yah later

**Comment by MSOB7YY** [comment]
some suggestions made by @Ahdi6262 at #731
> I am frequently frustrated when lyrics downloaded by Namida (from any source) contain "errors in text or incorrect timing (timestamps)". Currently, there is no way to correct these inaccuracies directly within the app, meaning I am forced to view incorrect lyrics repeatedly or manually create and manage local LRC files outside of Namida's ecosystem. The user has a passive role in correcting and customizing the lyrics data.
> 
> I would like to request an "In-App Lyrics Editing Feature" that allows users to modify and save lyrics (both text and synchronized timestamps) for individual tracks.
> 
> This feature should include:
>     1. An "Edit Lyrics" action/button available from the lyrics screen.
>     2. An editor that allows "modification of lyric text and corresponding timestamps" (compatible with the LRC format).
>     3. A saving mechanism that "saves the user-edited lyrics as the new local source" for that specific track.
>     4. The ability to set the "Lyrics Source" priority to "Local" to ensure that once lyrics are edited, the corrected version is always displayed instead of a re-downloaded, incorrect version from an online database.
>     * External File Editing:I currently have to find the lyrics
>       (or create a new LRC file), edit it manually in a separate text editor, and place it in the correct directory and the auto selection do not prioritize local . This is cumbersome, time-consuming, and does not seamlessly integrate with Namida's lyric fetching logic.
>     * Requesting Upstream Changes:" While I can suggest corrections to the original lyric sources , this is a slow process that doesn't guarantee a fix, and I would still prefer a faster, local customization option.
> 
> 
> The implementation could leverage the existing cache system. When lyrics are downloaded from a database, they are saved to a cache file may be . The 'Edit' function could allow users to modify this cached file, and upon saving, permanently designate this modified version as the high-priority "Local" source for the track. This would be a game-changer for users who value accuracy and customization in their music experience.
> 
> So as many music player don't have this systems to locally editing the lyrics inside the player this wood be quality feature that will increase popularity as the friction is reduced
